cli/lsp/resolver.rs

```diff
@@ -997,7 +997,7 @@ impl<'a> deno_graph::source::Resolver for SingleReferrerGraphResolver<'a> {
   ) -> Option<String> {
     self
       .jsx_import_source_config
-      .and_then(|c| c.default_specifier.clone())
+      .and_then(|c| c.import_source.as_ref().map(|s| s.specifier.clone()))
```

Comment:
I kind of feel like we should be providing the base to deno_graph, but I guess people don't use jsxImportSource with relative urls.

Reply:
yeah, I don't think other transpilers support relative paths and just mindlessly insert the given string value. Would be kinda cool for prototyping if relative-style import source would be supported though.